cli/graph_util.rs

```diff
@@ -150,7 +150,7 @@ pub fn graph_lock_or_exit(graph: &ModuleGraph, lockfile: &mut Lockfile) {
 }
 
 pub async fn create_graph_and_maybe_check(
```

Comment:
I think we should leave this method as is and add additional `create_graph_with_side_modules_and_maybe_check`. @dsherret thoughts on this one? I remember you changed this API recently to have a single root module.

Reply:
@dsherret any thoughts here?

Reply:
I feel like this is fine and preferred? What's the reason to have a separate function to restrict it to a single root module here?